templates/guests_table.php

```diff
@@ -2,81 +2,128 @@
-                
+
                 <?php if($guest->errors) echo '<br /><span class="errors">'.implode(', ', array_map(function($type) {
-                    return Lang::tr('recipient_error_'.$type);
-                }, array_unique(array_map(function($error) {
-                    return $error->type;
-                }, $guest->errors)))).'</span>' ?>
+                        return Lang::tr('recipient_error_'.$type);
+                    }, array_unique(array_map(function($error) {
+                        return $error->type;
+                    }, $guest->errors)))).'</span>' ?>
             </td>
-            
-            <td class="subject">
-                <?php if(strlen($guest->subject) > 15) { ?>
-                <span class="short"><?php echo Template::sanitizeOutput(mb_substr($guest->subject, 0, 15)) ?></span>
-                <span class="clickable expand">[...]</span>
-                <div class="full"><?php echo Template::sanitizeOutput($guest->subject) ?></div>
-                <?php } else echo Template::sanitizeOutput($guest->subject) ?>
-            </td>
-            
-            <td class="message  d-none d-lg-table-cell">
-                <?php if(strlen($guest->message) > 15) { ?>
-                <span class="short"><?php echo Template::sanitizeOutput(mb_substr($guest->message, 0, 15)) ?></span>
-                <span class="clickable expand">[...]</span>
-                <div class="full"><?php echo Template::sanitizeOutput($guest->message) ?></div>
-                <?php } else echo Template::sanitizeOutput($guest->message) ?>
-            </td>
-            
-            <td class="created d-none d-lg-table-cell"><?php echo Utilities::formatDate($guest->created) ?></td>
-            
-            <td class="expires" data-rel="expires">
+
+            <td class="expires" data-rel="expires" data-label="{tr:expiration}">
                 <?php echo $guest->getOption(GuestOptions::DOES_NOT_EXPIRE) ? Lang::tr('never') : Utilities::formatDate($guest->expires) ?>
             </td>
-            
-            <td class="actions">
+
+            <td class="guest_transfers" data-label="{tr:guest_transfers}">
+                <?php
+                    $guestTransfers = Transfer::fromGuest($guest);
+                    if (!count($guestTransfers)) {
+                ?>
+                    <ul class="fs-list fs-list--inline">
+                        <li>
+                            0
+                        </li>
+                        <li>
+                            <i class="fa fa-exclamation-circle"></i>
+                        </li>
+                    </ul>
+                <?php } else {
+                    $dc = count($guestTransfers->downloads);
+                    echo $dc;
+                } ?>
+            </td>
+
+<!--            <td class="subject">-->
```

Comment:
Is this commented out code still needed?

Reply:
This field was in the prototype but it is not in the backend so I left it commented.